src/RoomClient.ts

```diff
@@ -283,10 +289,13 @@ export class RoomClient {
     return this.createListLocally(name);
   }
 
-  private createMapLocally<T extends any>(name: string) {
+  private createMapLocally<T extends MapObject>(name: string) {
     const bus = new LocalBus<DispatchDocCmdMsg>();
     bus.subscribe((body) => {
-      this.dispatchMapCmd(name, body);
+      const client = this.mapClients[name];
```

Comment:
ah this is what was causing the double-apply of local updates? 🤔

Reply:
Yeah! :D